views/homepage.ejs

```diff
@@ -19,7 +19,17 @@
           >
             <img
               class="signup-download-button-img"
-              src="/images/app-page/LaunchPage-AppleStoreBadge-Mobile.png"
+              src="/images/app-badges/apple-store-badge-with-border.png"
+            />
+          </a>
+          <a
+            class="signup-download-button-link"
+            href="https://play.google.com/store/apps/details?id=com.shinetext.shine&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"
```

Comment:
Just confirming, is this the `pcampaignid` we're suppose to be using based on info we got from CW or AH?